src/test/java/htsjdk/samtools/cram/ref/EnaRefServiceTest.java

```diff
@@ -7,6 +7,8 @@
 
 import java.io.IOException;
 
+//calling this an FTP test so that it could fail in travis
+@Test(groups="ftp")
 public class EnaRefServiceTest extends HtsjdkTest {
```

Comment:
I don't like labelling this as an FTP test.  Maybe instead of a new SRA block we can have an EXTERNAL_APIS test type that covers SRA and ENA.

Reply:
done.